### DISCUSSION TITLE
epoch 1.15 Wallet hangs while trying to exit stake camillia.

### DISCUSSION THREAD

**Discussion by puchungotv** [tearust/teaproject#11] [General]
If you try to stake some camillia and input any CML ID for staking and exit the signature screen w/o actually entering your password. The wallet will hang at LOADING...... 

Easy fix is to refresh the browser.

**Comment by puchungotv** [comment]
the actual issue is not that section of the wallet is the communication between the polkadot extension and cancelling the request without entering the signature password and closing the extension.

**Reply by kevingzhang** [reply]
Yes. if you do not input the password, the wallet UI will just stupidly waiting forever. There is nothing we can do at this moment. Just hope the Polkadot extension can pop up much obviously.

**Comment by kevingzhang** [comment]
@puchungotv if you are satisfied with my answer, please click "This is the answer" so that my answer will be marked as an answer. thanks
